### DISCUSSION TITLE
Download failed: https://ghcr.io/v2/homebrew/core/libbluray/manifests/1.3.4

### DISCUSSION THREAD

**Discussion by sunsear** [Homebrew/discussions#4003] [Everyday usage] [answered]
Trying to install Freeswitch, which depends on libbluray. This seems unavailable, while the https://formulae.brew.sh/formula/libbluray#default page says that it should be available. 

```
MBP-van-Martin:master martinvandijken$ brew install libbluray
==> Downloading https://ghcr.io/v2/homebrew/core/libbluray/manifests/1.3.4
curl: (22) The requested URL returned error: 404

Error: libbluray: Failed to download resource "libbluray_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/libbluray/manifests/1.3.4
```
Running this on OSX Ventura 13.0.1 on Intel...

**Comment by sunsear** [comment]
Apparently this was a temporary error, it does now proceed.

**Comment by p-linnane** [comment, marked as answer]
https://github.com/Homebrew/homebrew-core/pull/116960

**Reply by sunsear** [reply]
Yup, that apparently fixed this one. Thanks for the update!
